projects/plugins/jetpack/extensions/README.md

```diff
@@ -277,6 +277,12 @@ Blocks can be registered but not available:
 - Registered: The block appears in the block inserter
 - Available: The block is included in the user's current plan and renders in the front end of the site
 
+## Block upgrade and deprecation paths
+
+When updating block markup or attributes, you will want to avoid block validation errors showing to all current users of the previous version of the block.
+
+In Jetpack the scenario that has been used historically is to create a deprecation. This [developer.wordpress.org](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-deprecation/) guide explains block deprecation in more detail. An example of block deprecation can be seen with the [Tiled Gallery block here](https://github.com/Automattic/jetpack/tree/7d97ced29dbbf6bd5c7a5d85bb5d752d9245f1c7/projects/plugins/jetpack/extensions/blocks/tiled-gallery) (see`index.js` in both the `deprecated` folder and the root `tiled-gallery` folder to see the `default` import and export in use).
```

Comment:
Seems a bit clearer to me to link more words to better describe what's being linked.
```suggestion
In Jetpack the scenario that has been used historically is to create a deprecation. [This developer.wordpress.org guide](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-deprecation/) explains block deprecation in more detail. An example of block deprecation can be seen with the [Tiled Gallery block here](https://github.com/Automattic/jetpack/tree/7d97ced29dbbf6bd5c7a5d85bb5d752d9245f1c7/projects/plugins/jetpack/extensions/blocks/tiled-gallery) (see`index.js` in both the `deprecated` folder and the root `tiled-gallery` folder to see the `default` import and export in use).
```
The second link there is more tricky. My first impulse would be to link only "here", but that's not very good practice. My second would be to link the whole first part of the sentence, "An example ... here", but that's maybe a bit big. What do you think?

Reply:
Good point for the first link, I added that in. With the second, I changed it to 'An example of block deprecation within Jetpack can be seen from [this `deprecated` directory within the Tiled Gallery block](https://github.com/Automattic/jetpack/tree/7d97ced29dbbf6bd5c7a5d85bb5d752d9245f1c7/projects/plugins/jetpack/extensions/blocks/tiled-gallery).' in 6684fa3ab520dc9260e215da6d9a9c72d7e4374b

Reply:
[](https://github.com/Automattic/jetpack/tree/7d97ced29dbbf6bd5c7a5d85bb5d752d9245f1c7/projects/plugins/jetpack/extensions/blocks/tiled-gallery)

Reply:
> With the second, I changed it to 'An example of block deprecation within Jetpack can be seen from [this deprecated directory within the Tiled Gallery block](https://github.com/Automattic/jetpack/tree/7d97ced29dbbf6bd5c7a5d85bb5d752d9245f1c7/projects/plugins/jetpack/extensions/blocks/tiled-gallery).' in https://github.com/Automattic/jetpack/commit/6684fa3ab520dc9260e215da6d9a9c72d7e4374b - how does that look?

Looks great to me!